### DISCUSSION TITLE
Pivot Table not ordering Dates correctly

### DISCUSSION THREAD

**Discussion by RandGitUser** [getredash/redash#6044] [Q&A]
Hi, I am now on the redash and I was able to figure out how to create a pivot table. However, I am not able to order dates in sequential order. In the attached image you can see that dates are ordered by the Day value, and not by the actual date. 

So, when the data is out side same month I get data that is not correctly sequenced.

Help would be greatly appreciated. 

![image](https://github.com/getredash/redash/assets/133785307/63e33a2c-fc27-4c3e-86bd-224c625a904a)

**Comment by manishtemkar** [comment]
Hello @RandGitUser, were you able to find a solution to the issue you mentioned earlier? I am experiencing the same problem with the ordering of integer values (Levels: 1-5) in my Pivot table.

**Reply by RandGitUser** [reply]
I got no update. I was hoping the devs would help but I am not seeing the reply with any insights.

**Reply by Smaran1** [reply]
i am also having the same issue, could not find a solution yet.

**Comment by edurojarod** [comment]
Im also facing this issue, no matter the format in the query there is no way to order correctly the pivot table. For some reason it default to MM/DD/YYYY format even if the query output is YYYY/MM/DD.

When you use the table viz you can change the format of the date. But looks like pivot is laking this feature.
